### ISSUE TITLE
Show application icon in `chrome://inspect`

### ISSUE THREAD

**Issue by jasta** [facebook/stetho#58]
There's a spot for the app icon to appear next to the process / app name in the `chrome://inspect` UI.  We should extract this from the manifest and display it.

**Comment by jasta** [comment]
Curse you Chrome!!!! https://code.google.com/p/chromium/issues/detail?id=461581

**Comment by jasta** [comment]
@paulirish looks like Chrome DevTools broke this recently as well.  favicon's no longer show up even for Chrome itself.

**Comment by paulirish** [comment]
Yup. Looks like a legit bug. Seems to be in our handler for obtaining favicons. It works through the chrome.debugger API but not when serviced over ADB

**Comment by facebook-github-bot-7** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by jasta** [comment]
Bump.  Bug remains upstream.
